supabase/migrations/20240517141758_add_is_delivery_bool_to_collection_centre_table.sql

```diff
@@ -0,0 +1,38 @@
+alter table "public"."collection_centres" add column "is_delivery" boolean not null default false;
+
+update "public"."collection_centres"
+       set is_delivery=true
+       where name='Delivery';
+       
```

Comment:
migration creates the 'is_delivery' column and updates the row where the name is 'Delivery' to be true; everything else defaults to false